x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/deployment/TrainedModelDeploymentTask.java

```diff
@@ -112,14 +112,15 @@ protected void onCancelled() {
     public void infer(Map<String, Object> doc, InferenceConfigUpdate update, TimeValue timeout, ActionListener<InferenceResults> listener) {
         if (inferenceConfigHolder.get() == null) {
             listener.onFailure(
-                ExceptionsHelper.conflictStatusException("[{}] inference not possible against uninitialized model", params.getModelId())
+                // inference not possible against uninitialized model"
```

Comment:
Is this comment necessary? If you think it is then please remove dangling double quote at the end